src/Compilers/Core/Portable/Operations/BasicBlock.cs

```diff
@@ -48,7 +48,7 @@ public BasicBlock(BasicBlockKind kind)
         /// <summary>
         /// PROTOTYPE(dataflow): During CR there was a suggestion to use different name - "Successor".
         /// </summary>
-        public (IOperation ReturnValue, Branch Branch) Next => InternalNext;
+        public (IOperation Value, Branch Branch) Next => InternalNext;
```

Comment:
Not related to this PR: I still don't get why have the distinction between `Next` and `InternalNext`. Couldn't we have `public (IOperation, Branch) Next { get; internal set; }`?

Reply:
This is a hierarchy of structures and we need an easy way to mutate individual fields within it. This cannot be done efficiently through a property.